### PR TITLE
[T5A1][T11-B2]Pankaj Bhootra

### PR DESCRIPTION
**Pull request by PankajB1997**

### CODE REVIEW

**Comment by PankajB1997** [comment]
Ready for review

File: doc/DeveloperGuide.md [hunk 1]
```diff
@@ -74,12 +83,48 @@ Use case ends.
...
+2a. The tag list is empty
+
+> Use case ends
+
+3a. The given index is invalid
+
+> 3a1. AddressBook shows an error message <br>
+  Use case resumes at step 2
+
+3b. The user enters 'cancel'
```
Comment (xdshang) [inline doc/DeveloperGuide.md]: This extention doesn't seem related to the 3rd case. Why would users type "cancel" when they were asked to type a new name for the tag.
Maybe you need to add more to the MSS, like "AddressBook asks for confirmation" and "User chooses to confirm or cancel", and then relate this extension to that part.
